compose/src/main/java/com/maplibre/compose/ramani/MapLibreCallbacks.kt

```diff
@@ -47,13 +64,32 @@ internal fun MapboxMap.setupEventCallbacks(
 
     onLongPressGestureCallback?.let {
         this.addOnMapLongClickListener { point ->
+            val screenLocation = projection.toScreenLocation(point)
+            val features = getFeaturesAtScreenLocation(screenLocation)
+
+            if (features.isNotEmpty()) {
+                Log.d("MapLibre", "Ignoring MapView long press gesture because feature exists. Use feature tap gesture instead.")
+                return@addOnMapLongClickListener false
+            }
+
             onLongPressGestureCallback?.invoke(
                 MapGestureContext(
+                    screenLocation,
                     MapGestureType.LONG_PRESS,
                     LatLng(point.latitude, point.longitude)
                 )
             )
             true
         }
     }
+}
+
+fun MapboxMap.getFeaturesAtScreenLocation(screenLocation: PointF): List<Feature> {
+    val features = queryRenderedFeatures(screenLocation)
+
+    if (features.isEmpty()) {
+        return emptyList()
+    }
+
+    return features
```

Comment:
It looks like the net effect is to just return the result of `queryRenderedFeatures` but it creates a fresh empty list in the case that the original list is already empty 🤔 Am I missing something?

Reply:
What we see here, is a function that I was going to originally transform within. But didn't need to in the end 🤣😅